src/components/fields/view/SpecifiedTimeViewer.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { get } from 'lodash-es';
+
+import { formatDateCustom } from '../../../utils/formatters';
+import Text from '../../Text';
+
+const formatMap = {
```

Comment:
It seems like you might could DRY this up by filtering on precisionMap from SpecifiedTimeEditor? As in, perhaps they could both rely on this similar object in a utils files instead?

Reply:
good call - done!